.github/ISSUE_TEMPLATE/permission_escalation_request_form.yaml

```diff
@@ -43,6 +43,8 @@ body:
       options:
       - label: Application (i.e. Glovebox admin)
       - label: Amazon Web Services
+      - label: Grafana
+      - label: Product Database Read Replica
```

Comment:
What if we did this:

* AWS - VPN + Console
* AWS - Grafana
* AWS - Postgres Read Replica

Reply:
All done.